packages/patternfly-3/patternfly-react/sass/patternfly-react/_dual-list-selector.scss

```diff
@@ -0,0 +1,127 @@
+.pf-dual-list-selector {
+  display: inline-block;
+  width: 230px;
+  min-width: 230px;
+  border: 1px solid #cecece;
+}
+
+.pf-dual-list-heading {
+  border-bottom: 1px solid #cecece;
+
+  .pf-dual-list-main-checkbox {
+    position: relative;
+    top: 3px;
+    left: 10px;
+  }
+
+  .pf-dual-list-filter {
+    margin-left: 20px;
+    margin-right: 15px;
+
+    input {
+      background-color: $color-pf-black-150;
+      border: 1px solid $color-pf-black-200;
+      width: 125px;
+      padding-left: 5px;
+      margin-top: 3px;
+    }
+  }
+
+  .pf-dual-list-filter .search-icon {
```

Comment:
Looks like anything with `.pf-dual-list-` can be moved to the top level instead of being nesting inside each other, reducing specificity.